src/sql/workbench/services/connection/browser/connectionController.ts

```diff
@@ -65,6 +65,7 @@ export class ConnectionController implements IConnectionComponentController {
 		tempProfile.groupFullName = '';
 		tempProfile.saveProfile = false;
 		tempProfile.azureAccount = azureAccount;
+		tempProfile.options.dstsToken = dstsToken;
```

Comment:
We're passing the token to the Kusto Tool Service the same way we pass the Azure token